api.json

```diff
@@ -125,6 +125,26 @@
             "description": "Safari: Password for safari_apns_p12 file",
             "writeOnly": true
           },
+            "apns_key_id": {
```

Comment:
Seems like formatting is little bit off, could you please move these fields to the left?